api_service.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+class ApiService:
```

Comment:
What's the different between a `Service` and `Client`? I thought this would be considered a client because it's the client side code used to communicate with an API. Either way works for me, just wondering.

Reply:
I chose the word `service` because it's agnostic to the caller, in the sense that you can use it as a  _service_ to communicate with our API.